<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login, Logout &amp; Register</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">font-size: 24px; font-weight: bold; color: #0275d8;</string>
      </property>
      <property name="text">
       <string>Perpustakaan Digital</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameInput">
      <property name="styleSheet">
       <string notr="true">padding: 8px; font-size: 14px; border: 1px solid #ccc; border-radius: 4px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordInput">
      <property name="styleSheet">
       <string notr="true">padding: 8px; font-size: 14px; border: 1px solid #ccc; border-radius: 4px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Enter your password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="styleSheet">
       <string notr="true">background-color: #5cb85c; color: white; padding: 10px; border: none; border-radius: 4px; font-size: 16px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="styleSheet">
       <string notr="true">background-color: #0275d8; color: white; padding: 10px; border: none; border-radius: 4px; font-size: 16px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #d9534f; color: white; padding: 10px; border: none; border-radius: 4px; font-size: 16px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
